features/machine/machine.feature

```diff
@@ -141,4 +141,38 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
-    
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I store the number of machines in the :num_to_restore clipboard
+    And admin ensures node number is restored to "<%= cb.num_to_restore %>" after scenario
+
+    Given I clone a machineset named "machineset-24363"
+    When I store the last provisioned machine in the :machine clipboard
+    And evaluation of `machine(cb.machine).node_name` is stored in the :noderef_name clipboard
+
+    Given I saved following keys to list in :taintsid clipboard:
+      | {"spec":{"taints": [{"effect": "NoExecute","key": "role","value": "master"}]}}  | |
+      | {"spec":{"taints": [{"effect": "NoSchedule","key": "role","value": "master"}]}} | |
```

Comment:
honestly this step is weird. I'd rather go update it to not require extra empty elements in table anymore. I'm not sure how it was introduced but that's not important. It is weird. In any case feel free to fix it or we can merge as is and fix step some day.

Reply:
It has been like this for such tables, we can merge I think and when the fix is available we can apply throughout